log/experimental_level/level.go

```diff
@@ -147,10 +63,87 @@ func (l *logger) Log(keyvals ...interface{}) error {
 	return l.next.Log(keyvals...)
 }
 
-func makeSet(a []string) map[string]struct{} {
-	m := make(map[string]struct{}, len(a))
-	for _, s := range a {
-		m[s] = struct{}{}
-	}
-	return m
+// Option sets a parameter for the leveled logger.
+type Option func(*logger)
+
+// AllowAll is an alias for AllowDebugAndAbove.
+func AllowAll() Option {
+	return AllowDebugAndAbove()
+}
+
+// AllowDebugAndAbove allows all of the four default log levels.
+func AllowDebugAndAbove() Option {
+	return allowed(levelDebug | levelInfo | levelWarn | levelError)
+}
```

Comment:
I'm relieved to see another acknowledgment that the order of these common levels is inconsistent in different libraries. I recall disagreeing with [_log15_'s ordering](https://github.com/inconshreveable/log15/blob/master/logger.go#L18-L22), which puts "critical" at the bottom/lowest and "debug" at the top/highest. "Severity" is an ordering that people will understand, but using function names like "InfoAndMoreSevere" didn't ring well to my ear.